sources/packages/web/src/views/aest/institution/ViewOfferingChangeRequest.vue

```diff
@@ -71,23 +98,62 @@ export default {
     },
   },
 
-  setup() {
+  setup(props: any) {
     const tab = ref("requested-change");
     const headerDetails = ref({} as ProgramOfferingHeader);
+    const offeringChangeApprovalStatus = ref(OfferingStatus.ChangeDeclined);
+    const assessOfferingChangeModalRef = ref(
+      {} as ModalDialog<OfferingChangeAssessmentAPIInDTO | boolean>,
+    );
+    const snackBar = useSnackBar();
+    const router = useRouter();
 
     //TODO: This callback implementation needs to be removed when the program and offering header component
     //TODO: is enhanced to load header values with it's own API call.
     const getHeaderDetails = (data: ProgramOfferingHeader) => {
       headerDetails.value = data;
     };
 
+    const assessOfferingChange = async (offeringStatus: OfferingStatus) => {
+      offeringChangeApprovalStatus.value = offeringStatus;
+      const responseData = await assessOfferingChangeModalRef.value.showModal();
+      if (responseData) {
+        try {
+          await EducationProgramOfferingService.shared.assessOfferingChangeRequest(
+            props.offeringId,
+            responseData as OfferingChangeAssessmentAPIInDTO,
+          );
+          const snackbarMessage =
```

Comment:
Can we move the `snackbarMessage` declaration right above the `snackBar.success`?